### DISCUSSION TITLE
Do not remove existing Workflows and channels on form import

### DISCUSSION THREAD

**Discussion by gjorgic** [dachcom-digital/pimcore-formbuilder#437] [Ideas]
Hi i notice that each import result with deletion of existing workflows and channels;

https://github.com/dachcom-digital/pimcore-formbuilder/blob/master/src/Tool/ImportExportProcessor.php#L126

Maybe i do not use form well, but this cause that (re)imported form stop working on any page i use before import, due to changing workflow ID; So i'm getting error that workflow is not defined;

Currently we patch importing logic and we just create missing workflows (_resolving by workflow name_) but i'm not sure what is side effect, and why you force deleting on each import?

If keeping existing workflows and channels is not so bad idea, maybe we can define "merging strategy" which can be defined through bundle configuration?

Thanks

**Comment by solverat** [comment]
Hey @gjorgic In our eyes, the form import is a "one time job" to setup a form on different environment for example (like from staging to production). 

So yes, the only way to solve this is to ask the user some questions, before importing the file (Like "Import Outputworkflows Yes/No?"). This could be a new feature for sure.

**Reply by solverat** [reply]
Ticket: https://github.com/dachcom-digital/pimcore-formbuilder/issues/466
